package.json

```diff
@@ -18,7 +18,7 @@
   "main": "app/index.js",
   "repository": "h5bp/generator-server-configs",
   "scripts": {
-    "test": "mocha --timeout 50000"
+    "test": "mocha --timeout 60000"
```

Comment:
As installing `h5bp` dep is one of the point of this PR, I thought that was a good idea to test if it was successful. If you guys think it's overkill, I can remove it by skipping the install in the tests.